src/App.tsx

```diff
@@ -15,36 +16,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_ALPHABET = 'Sort alphabetically',
+  SORT_BY_LENGTH = 'Sort by length',
+}
+
+type FilterParameter = {
+  sortField?: SortType | '';
+  isReversed?: boolean;
+};
```

Comment:
```suggestion
type FilterParameter = {
  sortField: SortType;
  isReversed: boolean;
};
```
Why did you do them optional, and an empty string you can add to a SortType enum as well